src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/reports/ZipFilesSummaryRepository.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.entity.reports;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.Envelope;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.UUID;
+
+public interface ZipFilesSummaryRepository extends JpaRepository<Envelope, UUID> {
```

Comment:
perhaps we don't need another envelope repository for reports. reports are mostly with extra features, multiple joins etc, so is this. simply injecting `EntityManager` to some sort of ReportsDAO would be good enough

Reply:
This will need some refactoring as we already have EnvelopeCountSummaryReport. I'll do it as part of cleanup

Reply:
yea, i believe so, either skimmed some prs or just got this idea now. happy to raise as a separate pr